### DISCUSSION TITLE
Changing download directories always makes the scraper crash. Any suggestions?

### DISCUSSION THREAD

**Discussion by revaDave** [UltimaHoarder/UltimaScraper#1363] [Q&A] [answered]
This maybe a very dumb question but hear me out:

In config.json I changed

"download_directories": [
          ".sites"

to

"download_directories": [
          "C:\Users\lockethal\Documents\scraped"

I even went as far to change all ".sites" to "C:\Users\lockethal\Documents\scraped"

What am I doing wrong?

Thank you for reading, and thank you for the wonderful program.

**Comment by UltimaHoarder** [comment, marked as answer]
Instead of single back slashes use double back slashes

**Reply by revaDave** [reply]
Worked like a charm! I knew it was something small that I was overlooking. I need to sleep more.
